File: experiment/java-python/test/j2pyrawpy/for__0__REARRANGE_POSITIVE_AND_NEGATIVE_NUMBERS_PUBLISH.py
#!/usr/bin/env python
""" generated source for module for__0__REARRANGE_POSITIVE_AND_NEGATIVE_NUMBERS_PUBLISH """
class X(object):
    """ generated source for class X """
    @classmethod
    def rearrange(cls, arr, n):
        """ generated source for method rearrange """
        i = -1
        temp = 0
        pos = i + 1
        neg = 0
        while pos < n and neg < pos and arr[neg] < 0:
            temp = arr[neg]
            arr[neg] = arr[pos]
            arr[pos] = temp
            pos += 1
            neg += 2

File: experiment_jp_test/mutants/for__0__REARRANGE_POSITIVE_AND_NEGATIVE_NUMBERS_PUBLISH.java
static void rearrange ( int arr [ ] , int n ) { int i = - 1 , temp = 0 ; int pos = i + 1 , neg = 0 ; while ( pos < n && neg < pos && arr [ neg ] < 0 ) { temp = arr [ neg ] ; arr [ neg ] = arr [ pos ] ; arr [ pos ] = temp ; pos ++ ; neg += 2 ; } }
